SCAutolib/models/file.py

```diff
@@ -144,6 +199,12 @@ def clean(self):
         except FileNotFoundError:
             logger.info(f"{self._conf_file} does not exist. Nothing to do.")
 
+    def backup(self):
```

Comment:
Well, I don't see `backup` method in SSSD class. So, this means that backup there is done in some other method of the SSSD class (what is not good) or somewhere else (what is also not very good). To avoid usage of this parent class method `SSSDConf` class only should implement its own method called `backup`. I didn't check this because I made changes only related to the controller by itself and didn't look for side effects because this work would be much bigger. From this perspective, I would suggest you add this method via PR. What do you think about this?